src/twisted/_version.py

```diff
@@ -7,5 +7,5 @@
 
 from incremental import Version
 
-__version__ = Version('Twisted', 17, 1, 0, dev=0)
+__version__ = Version('Twisted', 17, 1, 1, dev=0)
```

Comment:
I'm not sure if the next version of Twisted will be 17.2.0 or not, since the release process has changed: 

http://twistedmatrix.com/documents/current/core/development/policy/release-process.html#version-numbers

@hawkowl can clarify.